experimenter/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.6.0 and should not be changed by hand.
```

Comment:
So this is unrelated to this specific PR, but maybe worth discussing... This seems to go back and forth between <poetry version in repo> and <some local poetry version someone has> depending on whether the lockfile is generated in docker or locally. Is it worth trying to validate this in some way?

Reply:
Oh funny, look at that.  I think we can just rely on poetrys own enforcement of its lockfile.  There are cases where you can update the lockfile outside of the container in a way that will cause the container side to error out entirely.  I'd say if poetry is happy with the change inside the container then it's fine.

Reply:
Yea that's true -- the changes should be caught by existing testing so fair enough!